### PR TITLE
action: add comment

### PR DESCRIPTION
**Pull request by sileht**
Related #286

### CODE REVIEW

**Comment by sileht** [comment]
I merge it, but fill free to review it, I will fix your remark.

File: doc/source/actions.rst [hunk 1]
```diff
@@ -48,6 +48,28 @@ parameter:
 Once the backporting pull request is closed or merged, Mergify will
 automatically delete the backport head branch that it created.
 
+.. _comment action:
+
+comment
+=======
+
+The ``comment`` action closes the pull request without merging it.
```
Comment (jd) [inline doc/source/actions.rst]: really?

File: mergify_engine/actions/comment.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    def run(self, installation_id, installation_token, subscription,
+            event_type, data, pull, missing_conditions):
+        try:
+            pull.g_pull.create_issue_comment(self.config["message"])
+        except github.GithubException as e:
+            pull.log.error("fail to post comment on the pull request",
+                           status=e.status, error=e.data["message"])
+            return ("failure", "the close message can't be created", "")
```
Comment (jd) [inline mergify_engine/actions/comment.py]: "close message"?

"issue comment"

File: mergify_engine/actions/comment.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        return ("success", "The following message have been posted",
```
Comment (jd) [inline mergify_engine/actions/comment.py]: has been

**Comment by jd** [comment]
I also wonder if a test case that makes sure there's no "loop" of comment would be nice?

**Pull request merged**